src/Manager.js

```diff
@@ -1,27 +1,61 @@
 import User from './User'
 
 class Manager extends User {
-  constructor(date, rooms, bookings) {
-    super(id);
-    this.date = date;
+  constructor(customers, rooms, id, date, bookings) {
+    super(id, date, bookings);
+    this.customers = customers;
     this.rooms = rooms;
+    this.currentCustomer = {};
+  }
 
+  determineOccupiedRooms(date, bookingData) {
+    let roomsTaken = bookingData.reduce((filledRooms, bookedRoom) => {
+      if (bookedRoom.date === date) {
+        filledRooms.push(bookedRoom.roomNumber)
+      }
+      return filledRooms
+    }, []);
+    return roomsTaken;
   }
 
-  provideAvailableRooms(date) {
+  // want to match current date to booking date, if dates match, provide list of rooms that are not taken
+  provideAvailableRooms(date, bookingData, roomData) {
```

Comment:
I love the helper function stuff going on! (I see Elle's influence! That's awesome!)